src/modules/signer/signer.service.ts

```diff
@@ -139,8 +141,19 @@ export class SignerService {
         return result;
     }
 
+    /**
+     * @description Tries to create `eth_sign` conformant signature (https://eth.wiki/json-rpc/API#eth_sign)
+     * Signing method is determined based on previous signing
+     *
+     * @param message Message should have binary representation to avoid confusion of text with hexadecimal binary data
```

Comment:
```suggestion
     * @param message The message to be signed. The message should have binary representation to avoid confusion of text with hexadecimal binary data
```